src/elasticsearch/elasticsearch-sync.service.ts

```
@@ -0,0 +1,766 @@
+  async getUserDocument(userId: string): Promise<IUser | null> {
+    if (!isElasticsearchEnabled()) {
+      this.logger.debug(`Elasticsearch disabled, skipping get user document for: ${userId}`);
+      return null;
+    }
+
+    try {
+      this.logger.debug(`Getting user document for: ${userId}`);
+      const result = await this.userElasticsearchService.getUser(userId);
+      
+      // Convert Elasticsearch result to IUser format
+      if (result && result._source) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)